src/annotator/config/settings.js

```diff
@@ -10,10 +10,10 @@ function settingsFrom(window_) {
   var configFuncSettings = configFuncSettingsFrom(window_);
 
   /**
-   * Return the href URL of the first annotator link in the given document.
+   * Return the href URL of the first annotator sidebar link in the given document.
```

Comment:
The references to "annotator" here are hangovers from Annotator. In future perhaps we might want to provide some standard way to register an annotation client on the page. Until then I would be inclined to do away with the "type" attribute and make the "rel" attributes Hypothesis specific (eg. "hypothesis-sidebar", "hypothesis-client").

Before removing the "type" attribute for the "rel=sidebar" link however, we would need to modify the browser extension because it currently makes use of it when [unloading the client](https://github.com/hypothesis/browser-extension/blob/cc7a4a95338dbe21e8c261981cfa29ad25c69d8f/src/chrome/lib/destroy.js#L6).